.github/custom_setup_slurm_action/action.yml

```
@@ -0,0 +1,54 @@
+name: "setup-slurm-action"
+description: "Setup slurm cluster on GitHub Actions using https://github.com/galaxyproject/ansible-slurm"
+branding:
+  icon: arrow-down-circle
+  color: blue
+runs:
+  using: "composite"
+  steps:
+    # prior to slurm-setup we need the podmand-correct command
+    # see https://github.com/containers/podman/issues/13338
+    - name: Download slurm ansible roles
+      shell: bash -e {0}
+      # ansible-galaxy role install https://github.com/galaxyproject/ansible-slurm/archive/1.0.1.tar.gz
+      run: |
+        ansible-galaxy role install https://github.com/mila-iqia/ansible-slurm/archive/1.1.2.tar.gz
```

Do we need the commented line?

```suggestion
      # ansible-galaxy role install https://github.com/galaxyproject/ansible-slurm/archive/1.0.1.tar.gz
      run: |
        ansible-galaxy role install https://github.com/mila-iqia/ansible-slurm/archive/1.1.2.tar.gz
```

---

We don't need it, but it's the "default" ansible slurm image from the github action I'm using. I switched it to use our fork's version.